ports/esp32/sdkconfig.h

```diff
@@ -5,7 +5,7 @@
 
 #define CONFIG_TRACEMEM_RESERVE_DRAM 0x0
 #define CONFIG_BT_RESERVE_DRAM 0x0
-#define CONFIG_ULP_COPROC_RESERVE_MEM 0
+#define CONFIG_ULP_COPROC_RESERVE_MEM 2040
```

Comment:
Was there a reason for the value 2040?  Why not 2048?  (The default value from the ESP IDF is 512.)

Also, isn't in necessary to define ULP_COPROC_ENABLED?

Reply:
ULP_COPROC_ENABLED seems to be ignored by the current build system, but for consistency we should probably set it. I choose the largest posible value. 2048 is not posible because micropython it self uses a part of the rtc slow memory already. Sadly I found no way to calculate the remaining space there and had to hard code it.

Reply:
I think we should set ULP_COPROC_ENABLED to 1 (it seems to be used by esp_sleep_enable_ulp_wakeup).

For the RESERVE_MEM value: ultimately I think the best solution is to overlap the ULP reserved mem with the machine.RTC().memory() feature so that the RTC class can be used to read/write into the ULP and they can talk to each other.  So, for example, they'd both use the first 2048 bytes of RTC slow memory.  Then it's up to the user to decide how much is used by each one.  This can be done quite easily by forcing `rtc_user_mem_data` to be located at 0x50000000.  But this can be done later.  For now, 2040 is a good enough value for the RESERVE_MEM (but it may change in the future).